src/mig/modules/scribe/doc.rst

```diff
@@ -0,0 +1,148 @@
+===================================
+Mozilla InvestiGator: scribe module
+===================================
+:Author: Aaron Meihm <ameihm@mozilla.com>
+
+.. sectnum::
+.. contents:: Table of Contents
+
+The scribe module provides host-based analysis based on a JSON document
+containing a series of tests. The module is based on the scribe engine;
+scribe can be found `here <https://github.com/mozilla/scribe>`_.
+
+The scribe module is intended to help support:
+
+* Executing policy checks on systems, for example as part of using MIG for vulnerability management
+* Execute more advanced file content tests involving dependencies
+
+In addition to supporting on-host analysis using scribe documents, the
+module can also be used to query elements of the host system such as the
+package manager database for installed package information.
+
+This document does not discuss the details around writing scribe tests, the
+scribe project documentation should be reviewed for that. This document focuses
+on usage of the scribe module within MIG and provides some examples.
+
+Usage
+-----
+The scribe module supports two execution modes, scribe document analysis
+and package query mode. To use document analysis mode, the `path` option can
+be given to the module along with the path to a scribe document. To use
+package query mode, the `pkgmatch` option is supplied with a regular expression
+specifying which packages to return information for from the agent.
+
+Document analysis mode
+~~~~~~~~~~~~~~~~~~~~~~
+In document analysis mode, a JSON document is supplied containing a valid
+scribe document.
+
+A scribe document contains a series of tests. Tests are typically comprised
+of two elements - a test source, which details a source of information from
+the host system, and an evaluator which specifies criteria that must match
+for the test to be true.
+
+The following is a simple document example that validates OpenSSL is at least
+version 1.0.1e. If this test fails, it will return true.
```

Comment:
`If this test fails, it will return true.`
I'm bothered by this. If the test says "list everything that has openssl >= 1.0.1e", then the test should succeed when a host has openssl >= 1.0.1e, not the other way around.